### ISSUE TITLE
add a way to pick up skeletal remains

### ISSUE THREAD

**Issue by Spacedong** [d3athrow/vgstation13#10857]
they're a nightmare for the janitor, can't be pulled, put in locker ,or picked up

**Comment by ComicIronic** [comment]
Can't you just clean them?

**Comment by Blithering** [comment]
No, and the same goes for the massive amounts of random robot remains scattered around deff motmaints.

**Comment by Megamaw** [comment]
oh god, please, yes, this.
